tests/client_pulp3/test_client_distributions.py

```
@@ -60,3 +60,54 @@ async def test_create_distribution_with_repository_href(respx_mock, pulp3_client
+        distr = await client.get_distribution(
+            "prod-dist",
+        )
+
+        assert isinstance(distr, dict)
+        assert distr.pop("base_path") == "https://pulp.example.com/production/repo"
+        assert distr.pop("fake_data") == {}
+        assert not distr
+
+
+@pytest.mark.anyio
+async def test_get_distribution(respx_mock, pulp3_client):
```

Pylint (4.0.7)

[error] function already defined line 66

(E0102)